eclipsesource/vscode-memory-inspector
# Allow to configure word size and add group separator

#### What it does

* Introduces user option for bytes per word (VS Code settings & per view)
* Rename "Bytes per Group" into "Words per Group"
* Add right margin to groups

Fixes https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/issues/74
Fixes https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/issues/75

#### How to test
* Start debug session and open two memory view instances
* Verify there are separating spaces between groups
* Change bytes per word and check it is applied correctly to the memory data in the individual views
* Use the reset settings button in the options and verify they are reset correctly
* Change the VS Code settings regarding "Bytes per Word"
* Use the reset settings button again to check the new settings are applied

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
